AnyServe/AnyServe/Controllers/FileUploadController.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AnyServe.Models;
+using System.IO;
+using System.Threading.Tasks;
+using AnyServe.Storage;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using System.Net;
+using System.Text;
+using System.Collections;
+using AnyServe.Utility;
+
+namespace AnyServe.Controllers
+{
+    [Route("api/[controller]")]
+    public class MediaController : Controller
+    {
+        private IWebHostEnvironment _appEnvironment;
+        private ConstantString _helper;
+        public MediaController( IWebHostEnvironment appEnviroment)
+        {
+            _appEnvironment = appEnviroment;
+            _helper = new ConstantString();
+        }
+
+        [HttpGet]
+        public IActionResult GetAllFiles()
+        {
+            string pathToFiles = Path.Combine(_appEnvironment.WebRootPath, _helper.directoryName);
+
+            return Ok(AllFilesInWebRootPath(pathToFiles));
+        }
+
+        // Single file upload
+        [HttpPost("UploadFile")]
+        public async Task<IActionResult> UploadFile([FromForm] IFormFile uploadedFile)
+        {
+            if (uploadedFile != null && Path.GetExtension(uploadedFile.FileName) != null)
```

Comment:
not all files usually have extension for some OS, ok for now but let keep it in a mind